service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json

```diff
@@ -0,0 +1,761 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "clusterLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "Location of the Cluster"
+      }
+    },
+    "clusterName": {
```

Comment:
Since this is a DNS name, the only way to truly figure out if the selected cluster name will go through is by deploying it, so adding unique does not really reduce the chances that it the deployment will go through or not. So wont fix this. This parameter name is carried though out the product, so making a cosmetic change here may not be the right thing to do anyway.


From: Kay Singh [mailto:notifications@github.com]
Sent: Tuesday, December 1, 2015 2:41 PM
To: Azure/azure-quickstart-templates <azure-quickstart-templates@noreply.github.com>
Cc: Chacko Daniel <chackdan@microsoft.com>
Subject: Re: [azure-quickstart-templates] WAD 5 Node (#863)


In service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json<https://github.com/Azure/azure-quickstart-templates/pull/863#discussion_r46351049>:

> @@ -0,0 +1,761 @@

> +{

> +  "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",

> +  "contentVersion": "1.0.0.0",

> +  "parameters": {

> +    "clusterLocation": {

> +      "type": "string",

> +      "metadata": {

> +        "description": "Location of the Cluster"

> +      }

> +    },

> +    "clusterName": {

If this needs to be unique, then it'd be good to prefix this parameter with a "unique" prefix to set the right expectations

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-quickstart-templates/pull/863/files#r46351049>.